lib/editor/util/validation.js

```diff
@@ -36,8 +37,14 @@ export function validate (
   const valueDoesNotExist = doesNotExist(value)
   const isRequiredButEmpty = required && valueDoesNotExist
   const isOptionalAndEmpty = !required && valueDoesNotExist
-  let reason = 'Required field must not be empty'
   const agencies = getTableById(tableData, 'agency')
+  let locationType: ?number = 0
+  let reason = 'Required field must not be empty'
+
+  // entity.locationtype is a string.  Convert to number for conditinals later on.
+  if (entity && entity.location_type) {
+    locationType = parseInt(entity.location_type)
```

Comment:
Always supply the `radix` argument to `parseInt`. Learn more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#octal_interpretations_with_no_radix).